Entities/Attachable.cpp

```diff
@@ -125,7 +125,10 @@ namespace RTE {
 		} else if (propName == "InheritedRotAngleRadOffset" || propName == "InheritedRotAngleOffset") {
 			reader >> m_InheritedRotAngleOffset;
 		} else if (propName == "InheritedRotAngleDegOffset") {
+			const char* locale = std::setlocale(LC_ALL, nullptr);
```

Comment:
This should be `const char *locale...`. Applies everywhere, obviously. I'm happy keeping it const since editing the pointed to chars is apparently undefined behaviour.

That said, I'm not sure you need this? Can't you just call `std::setlocale(LC_ALL, "");` to reset the locale back to its default later? Nbd either way, but I'm curious.